### ISSUE TITLE
I WANT A WEBSITE!

### ISSUE THREAD

**Issue by radioblahaj** [Mountio/Site-Suggestions#1]

**Comment by AmazingMech2418** [comment]
OK. Come back once the repo is done.

**Comment by radioblahaj** [comment]
:)
K

**Comment by radioblahaj** [comment]
we should makes users sign up and be able to log in!

**Comment by AmazingMech2418** [comment]
We should! We'll need PHP though. The repo is done!

**Comment by AmazingMech2418** [comment]
@redmangospros Repo done! Make a new issue to get your site!

**Comment by radioblahaj** [comment]
YAY! Php, we can do that!

**Comment by AmazingMech2418** [comment]
OK. So, do you want a site?

**Comment by radioblahaj** [comment]
No i own this lol

**Comment by AmazingMech2418** [comment]
Co-own. We both own it, but you created it.

**Comment by radioblahaj** [comment]
😀

**Comment by radioblahaj** [comment]
We are not cpmpteing with swiftsites, we can be better.

**Comment by AmazingMech2418** [comment]
Yeah. We will be better. We also have gaming and other software!

**Comment by radioblahaj** [comment]
I know, so cool, But it comes to the look.

**Comment by AmazingMech2418** [comment]
Yeah. We need better themes.

**Comment by radioblahaj** [comment]
What Shiuld we do? Hire attpikachu?
Look at my scratch profile

**Comment by AmazingMech2418** [comment]
I already invited him/her.

**Comment by redmangospros** [comment]
Are you him or her?

**Comment by amazinigmech2418** [comment]
If you mean @aatpicchu, no. If you mean am I a him or her, why do you need to know that?

**Comment by radioblahaj** [comment]
Sorry
